src/generator/www/js/main.js

```diff
@@ -0,0 +1,52 @@
+$(document).ready(function() {
+
+  let $email =$("#email");
+  let $name  =$("#name"); 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Infix operators must be spaced.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711222409/issues/source?bid=3233857&fileBranchId=3344140#l4)